src/debugger.ts

```diff
@@ -121,41 +120,45 @@ export class Debugger
     this.subscriptions.forEach((subscription) => subscription.dispose());
   }
 
+  private getSockets(): string[] {
```

Comment:
When getting sockets, we should take users' `launch.json` configuration into account, especially the `env`. When users specifying things like `RUBY_DEBUG_SOCK_DIR` when starting the debuggee process, the `rdbg --util=list-socks` call needs to have the same value or it wouldn't find anything.

The `vscode-rdbg` extension [takes the configuration into account](https://github.com/ruby/vscode-rdbg/blob/master/src/extension.ts#L407C28-L412) so it supports the above use case.